### PR TITLE
Rename `--brokers` to `--bootstrap-servers`

### LINKED ISSUE #232: Rename `--brokers` to `--bootstrap-server`
This is used by all Kafka CLI tools

### PR DESCRIPTION
**Pull request by philipp94831**
Fixes #232

### CODE REVIEW

File: charts/streams-app/templates/deployment.yaml [hunk 1]
```diff
@@ -115,9 +115,9 @@ spec:
             - name: "{{ .Values.configurationEnvPrefix }}_VOLATILE_GROUP_INSTANCE_ID"
               value: "true"
           {{- end }}
-          {{- if hasKey .Values.streams "brokers" }}
-            - name: "{{ .Values.configurationEnvPrefix }}_BROKERS"
-              value: {{ .Values.streams.brokers | quote }}
+          {{- if hasKey .Values.streams "bootstrapServers" }}
+            - name: "{{ .Values.configurationEnvPrefix }}_BOOTSTRAP_SERVER"
```
Comment (philipp94831) [inline charts/streams-app/templates/deployment.yaml:119]: The cli param for all kafka cli tools is always `--bootstrap-server`. However it is `StreamsConfig.BOOTSTRAP_SERVERS_CONFIG`. Thats why I named it bootstrapServers in the chart. wdyt?

Comment (torbsto) [reply charts/streams-app/templates/deployment.yaml:119]: I would prefer if we're consistent and then stick to using the property names everywhere, so `bootstrap.servers` in this case.

Comment (philipp94831) [reply charts/streams-app/templates/deployment.yaml:119]: So you would use `--bootstrap-servers`?

Comment (torbsto) [reply charts/streams-app/templates/deployment.yaml:119]: yes

Comment (philipp94831) [reply charts/streams-app/templates/deployment.yaml]: I added an alias to have both available

File: README.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ public class StreamsBootstrapApplication extends KafkaStreamsApplication {
 
 The following configuration options are available:
 
-- `--brokers`: List of Kafka brokers (comma-separated) (**required**)
+- `--bootstrap-servers`, `--bootstrap-server`: List of Kafka bootstrap servers (comma-separated) (**required**)
```
[the remark this answers is not in the document]
Comment (philipp94831) [reply README.md:103]: Did you see this conversation? https://github.com/bakdata/streams-bootstrap/pull/242#discussion_r1687819037 I don't think it is confusing tbh. In the helm chart there is only one option (bootstrapServers) and the singular alias just exists to use the same as kafka CLI tools. Both have exactly the same effect

**Pull request merged**